README.md

```diff
@@ -1,49 +1,84 @@
-nomad-skeleton-driver-plugin
+Nomad Skeleton Driver Plugin
 ==========
 
-This project is a [Hashicorp Nomad](https://www.nomadproject.io/) [task driver
-plugin](https://www.nomadproject.io/docs/drivers/index.html) skeleton
-implementation. You can clone and modify this to get started writing your own
-task driver plugin.
+Skeleton project for
+[Nomad task driver plugins](https://www.nomadproject.io/docs/drivers/index.html).
+
+This project is intended for bootstrapping development of a new task driver
+plugin.
+
+- Website: [https://www.nomadproject.io](https://www.nomadproject.io)
+- Mailing list: [Google Groups](http://groups.google.com/group/nomad-tool)
 
 Requirements
 -------------------
 
-- [Nomad](https://www.nomadproject.io/downloads.html) v0.9.5+
-- [Go](https://golang.org/doc/install) v1.12.x (to build the plugin)
+- [Nomad](https://www.nomadproject.io/downloads.html) v0.9+
+- [Go](https://golang.org/doc/install) v1.11 or later (to build the plugin)
```

Comment:
these tested fine with 1.11.x?

Reply:
Yes, I tested on Go 1.10, 1.11, 1.12 and 1.13.

1.10 failed due to lack of modules support.

Reply:
👍